app/services/webhook_service.rb

```diff
@@ -0,0 +1,22 @@
+require 'rest-client'
+
+class WebhookService
+
+  def self.publish!(webhook:, event:)
+    return false unless webhook.event_types.include? event.kind
```

Comment:
Only publish if this event's kind is in the list of event kinds for this webhook.